### PR TITLE
Allow overriding notifier host

### PR DESCRIPTION
**Pull request by adamnicholson**

### CODE REVIEW

File: src/Notifier.php [hunk 1]
```diff
@@ -22,10 +22,11 @@ class Notifier
      */
     public function __construct($opt = [])
     {
-        // TODO: test that projectId and projectKey exists
-        $this->opt = array_merge($opt, [
+        $this->opt = array_merge([
             'host' => 'api.airbrake.io',
-        ]);
+            'projectKey' => '',
+            'projectId' => ''
```
Comment (vmihailenco) [inline src/Notifier.php:28]: Remove `projectId` and `projectKey`?

File: src/Notifier.php [hunk 2]
```diff
@@ -22,10 +22,11 @@ class Notifier
      */
```
Comment (vmihailenco) [inline src/Notifier.php:22]: Please update doc comment.

**Comment by vmihailenco** [comment]
I assume this is for errbit? Then you can probably add an example to the readme for errbit users?

**Comment by md5** [comment]
@adamnicholson Do you plan on updating this PR to address the things pointed out by @vmihailenco? I'd like to see this change as well in order to support Errbit as @vmihailenco surmised.

**Comment by kyrylo** [comment]
Ping @adamnicholson

**Comment by vmihailenco** [comment]
Similar functionality was added in https://github.com/airbrake/phpbrake/pull/33.

**Pull request closed without merging**